addons/account_edi_ubl_bis3/models/account_edi_format.py

```diff
@@ -102,6 +102,22 @@ def grouping_key_generator(tax_values):
 
         return values
 
+    ####################################################
+    # Account.edi.format override
+    ####################################################
+
+    def _check_move_configuration(self, invoice):
+        errors = super()._check_move_configuration(invoice)
+        edi_format = invoice.journal_id.edi_format_ids
+        edi_format_required = any(ef._is_required_for_invoice(invoice) for ef in edi_format)
+        is_edi_needed = invoice.is_invoice(include_receipts=False) and edi_format_required
+        if is_edi_needed:
```

Comment:
This check should only be applied on relevant bis3 edi.formats so you need to define an additional method telling which format needs to be checked with it
```py
if self.code in self._get_bis3_edi_format_codes(): # this method doesn't exist yet
    # blablabla
```